integration-tests/jail/docker-compose.yml

```diff
@@ -20,6 +23,9 @@ services:
     volumes:
       - ./disk:/root/disk
       - ./disk/config1:/root/config
+      - /nix:/nix
+    devices:
+      - /dev/isgx
```

Comment:
the device part isn't portable, as e.g. on azure it's `/dev/sgx`

Reply:
i will modify it

Reply:
on ubuntu  std installation, it is /dev/isgx,  i think azure is old style, but i will change to azure paradigm.

Reply:
also for docker, it's coded before using docker-compose, i will move it to docker-compose

Reply:
for docker, i'll change it.

Reply:
you can just use an environment variable (and default it to /dev/isgx)... just like in the existing integration tests @calvinaco wrote -- have you managed to read them?

Reply:
https://github.com/crypto-com/chain/blob/master/integration-tests/docker-compose.yml#L25

Reply:
https://github.com/crypto-com/chain/blob/master/integration-tests/const-env.sh#L41

Reply:
but I think there may be a problem in the existing integration tests that they do `/dev/something:/dev/else` -- for example, if you expose `/dev/isgx:/dev/sgx` (or `/dev/sgx:/dev/isgx`), it may not work